docker-compose.yaml

```diff
@@ -0,0 +1,55 @@
+version: '3.1'
+services:
+  api:
+    image: node
```

Comment:
We should maybe add a version number to node image (at least for the API)?

Reply:
Yes you are right but which one ?

Reply:
LTS should work